modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/GodotPluginsInitializerGenerator.cs

```diff
@@ -24,10 +24,13 @@ public void Execute(GeneratorExecutionContext context)
 
 namespace GodotPlugins.Game
 {
-    internal static partial class Main
+    public static partial class Main
     {
+        /// <summary>
+        /// Should not be touched endless you know what your doing this is a UnmanagedCallersOnly function
```

Comment:
**unless** you know know what your doing

Reply:
if we're going to have a grammar fix, might as well go the whole way:

**unless** you know know what **you are** (or **you're**, not ~~your~~) doing **. (period)** This is is a UnmanagedCallersOnly function
```suggestion
        /// Should not be touched unless you know what you are doing. This is a UnmanagedCallersOnly function.
```